Modules/Core/SpatialObjects/include/itkSpatialObjectToImageFilter.hxx

```diff
@@ -164,6 +166,17 @@ SpatialObjectToImageFilter<TInputSpatialObject, TOutputImage>::GetSpacing() cons
   return m_Spacing;
 }
 
+template <typename TInputSpatialObject, typename TOutputImage>
+const typename TOutputImage::SpacingType &
+SpatialObjectToImageFilter<TInputSpatialObject, TOutputImage>::GetSpacingVector() const
+{
+  for (unsigned int i = 0; i < OutputImageDimension; ++i)
+  {
+    m_SpacingVector[i] = m_Spacing[i];
```

Comment:
If this assignment is done in the getter, what is the point of also having it as an ivar? This could be a local variable in the getter instead. Same remark for the other one.

Reply:
I was trying to avoid returning a reference to a local var and to make the minimal changes to the code to preserve backward compatibility.   What do you recommend?

Reply:
Aha, so we can't return spacing by value? If so, this is fine.

Reply:
Yeah - this is still using a very old (the original) ITK API where spacing was double * - which doesn't wrap for python, hence this PR.  For ITK v6.0, I'd like to do a major update to SpatialObjects to follow ITK's current API, but it will break backward compatibility.